### PR TITLE
QA - Frame work

### PR DESCRIPTION
**Pull request by Prav-Patel**

### CODE REVIEW

File: frameworks/engineering/quality.md [hunk 1]
```diff
@@ -12,19 +12,22 @@ topics:
...
+        - "Transparent about mistakes they've made"
         - "Accepts feedback graciously and looks to action feedback"
         - "Gives feedback to peers when asked"
-        - Communicates clearly to other disciplines and within squad with little need for clarification
+        - Communicates clearly to other disciplines and within team with little need for clarification
+        - "Can successfully get buy-in for their proposals, from their team"
       - level: 3
         criteria:
-          - "Transparent about mistakes they've made, early"
+          - "Transparent about mistakes they've made, early informs the people"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:30]:
```suggestion
          - "Transparent about mistakes they've made, informs relevant people as soon as possible."
```

File: frameworks/engineering/quality.md [hunk 2]
```diff
@@ -12,19 +12,22 @@ topics:
     content:
       - level: 1
         criteria:
-        - "Provides regular status updates to their squad and discipline"
+        - "Provides regular status updates to their team and discipline"
         - "Points out improvements in test scenario reviews"
+        - "Contributes in three Amigos"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:17]: Do we call them three amigos or kickoffs?

Be good to be consistent across the ranges if possible.

File: frameworks/engineering/quality.md [hunk 3]
```diff
@@ -38,85 +41,169 @@ topics:
...
+          - "Can consider different approaches to testing a feature"
+          - "Understands our company goals and makes decisions to help support the bigger picture"
+
   - name: "leadership"
     title: "👩‍💼 Leadership"
     content:
+      - level: 2
+        criteria:
+          - "Runs delivery team Stand-ups"
+          - "Runs show and tells to stake holders"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:75]: Bit of an interesting one this. Most of our stakeholders aren't engaged with the team on a daily level, and show and tells are done by PO / Product.

File: frameworks/engineering/quality.md [hunk 4]
```diff
@@ -38,85 +41,169 @@ topics:
...
+        - "Keeps documentation up to date"
+        - "Collaborate with others on elaborating work"
+        - "Good knowledge of the other teams/areas of R&D"
       - level: 3
         criteria:
         - "Provides valuable input to RFCs from other testers"
         - "Proactively improves test frameworks and tools they encounter, 'this doesn't make sense, I'm going to do something about it'"
         - "Contributes to scaling testing hiring (e.g. leads calls, onsite interviews)"
         - "Builds simple test tools, harnesses, frameworks for the benefit of all testers"
+        - "Lead retros and other team improvement sessions"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:119]: I honestly think anyone should be able to do this.

File: frameworks/engineering/quality.md [hunk 5]
```diff
@@ -38,85 +41,169 @@ topics:
...
     title: "✨ Influence"
     content:
       - level: 1
         criteria:
           - "Improves documentation that is incorrect"
+          - "Contributes to retro discussions"
+          - "Clearly announces standup updates"
       - level: 2
         criteria:
         - "Proactively raises issues they spot in retrospectives"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:107]: As below.

File: frameworks/engineering/quality.md [hunk 6]
```diff
@@ -38,85 +41,169 @@ topics:
...
           - "Delivers large well-defined tasks and solves small scope not-well-defined problems"
-          - "Contributes to writing proposals (Co-authors with more experienced Engineer)"
+          - "Contributes to writing test proposals (Co-authors with more experienced Engineer)"
           - "Identifies areas of testing within the business that can be improved and suggests improvements"
           - "Breaks down large problems into smaller iterative steps"
+      - level: 4
+        criteria:
+          - "Able to diagnose or reproduce issues with limited information"
+          - "Supports implementation of a feature and any issues that may arise from it"
+          - "Makes pragmatic choices on their testing approach"
```
Comment (Paul2e) [inline frameworks/engineering/quality.md:64]: This needs to be in BOLD, SIZE 50!

**Review by Paul2e — CHANGES_REQUESTED** [review]

**Comment by Paul2e** [comment]
@Prav-Patel Mostly there, just a few comments. I'll leave it upto you if you want to make them. 👍

File: package.json [hunk 7]
```diff
@@ -5,39 +5,39 @@
   "description": "BrightHR progression framework site",
   "author": "Alex Maher<alex.maher@brighthr.com> Prav Patel<prav.patel@brighthr.com> Paul Touhey<paul.touhey@brighthr.com>",
   "dependencies": {
-    "@babel/polyfill": "^7.0.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.12",
-    "@fortawesome/free-brands-svg-icons": "^5.6.3",
-    "@fortawesome/react-fontawesome": "^0.1.3",
-    "@reach/router": "^1.2.1",
-    "babel-plugin-styled-components": "^1.10.0",
+    "@babel/polyfill": "^7.12.1",
```
Comment (Paul2e) [inline package.json:8]: 👍

File: frameworks/engineering/quality.md (hunk 1 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:30]: 👨🏽‍🏭 Fixed

File: frameworks/engineering/quality.md (hunk 2 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:17]: 👨🏽‍🏭 Fixed
**Definitions**
**3 Amigos** - Would be for a ticket basis within a feature - BA/QA/DEV/UX. making sure we all clear on what needs to be done.
**Kick Off** - would be the start of a project the Whats, Whys Whos - Any high level designs or requirements shared based on a Problem we need to resolve.

Does that make sense?

Comment (Prav-Patel) [reply frameworks/engineering/quality.md:17]: Ahh I see what you are saying here.... 
Shall I update to both to:

"Contributes in Kick Offs"
"Points out improvements in test scenario reviews or in three amigo sessions"

- does that sound better?

File: frameworks/engineering/quality.md (hunk 3 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:75]: We dont have a PO so we have historically ran these sessions with the Dev Lead.
Today for example - Mahmud ran Payroll Navigator to Senior Stakeholders, Last Mukesh did it.
Friday Big Stand- ups also count in this scenario I think. 
They might not need to do it, but if they have the skill to then i think that is warrants merit. Dont you agree?

File: frameworks/engineering/quality.md (hunk 4 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:119]: as in all levels? or all disciplines?

File: frameworks/engineering/quality.md (hunk 6 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:64]: Preach 🙌🏽

File: frameworks/engineering/quality.md (hunk 2 above)
Comment (Prav-Patel) [reply frameworks/engineering/quality.md:17]: @Paul2e What do you think?

Comment (Prav-Patel) [reply frameworks/engineering/quality.md:17]: I've gone with adding them both.

**Review by Paul2e — APPROVED** [review]

**Pull request merged**